set/set.go

```diff
@@ -383,12 +395,28 @@ func (s *Set) entryStatusToSetCounts(entry *Entry) {
 
 		if entry.Attempts >= AttemptsToBeConsideredFailing {
 			s.Status = Failing
+
+			return s.messageError("has failed uploads")
 		}
 	case Missing:
 		s.Missing++
 	case AbnormalEntry:
 		s.Abnormal++
 	}
+
+	return nil
+}
+
+func (s *Set) messageError(msg string) error {
+	return s.createAndSendMessage("🟥", msg)
```

Comment:
These strings should definitely be in consts, and it'd be hard to add another 'error' log without copying and pasting the string.